spec/xcpretty/formatters/simple_spec.rb

```diff
@@ -95,6 +95,11 @@ module XCPretty
         "    P _tupleByAddingObject__should_add_a_non_nil_object [PENDING]"
       end
 
+      it "formats measuring tests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.